README.md

```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
 This repository contains the reference implementation of the OpenYOLO protocol,
 and the standard API and Service Provider Interface (SPI).
 
-OpenYOLO is inspired by, and ultimately intended to replace, Google's
-[Smart Lock for Passwords][yolo] API, which is
-internally known as YOLO (You Only Login Once).
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/445775/30930082-60d21ace-a375-11e7-90cf-cc75fe5747fd.gif"/>
```

Comment:
You should be able to Markdown-ify this image:

```
![Animation demonstrating auto sign-in](https://user-images.githubusercontent.com/445775/30930082-60d21ace-a375-11e7-90cf-cc75fe5747fd.gif)
```

Reply:
I had tried a few times, but was not able to also center the image.

Reply:
You worry too much about formatting :rofl: